src/marp-cli.ts

```diff
@@ -309,6 +309,7 @@ export default async function(argv: string[] = []): Promise<number> {
     if (!(e instanceof CLIError)) throw e
 
     cli.error(e.message)
+    process.exit(e.errorCode)
```

Comment:
Because marp-cli.js does `.then(exitCode => process.on('exit', () => process.exit(exitCode)))`, current code never exit process in this case.